source/fundamentals/transactions.txt

```diff
@@ -127,6 +128,39 @@ A ``Session`` also has methods to retrieve session
 properties and modify mutable session properties. View the :ref:`API
 documentation <api-docs-transaction>` to learn more about these methods.
 
+.. _golang-session-txn-options:
+
+Session and Transaction Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can set options at the session level and the transaction level to
+customize how the driver performs a transaction. The following steps
+describe how to set options for all transactions run within a given
+``Session``:
+
+1. Create a ``TransactionOptions`` instance. You can specify options
+   such as a write concern, read concern, and read preference for all
+   transactions run in a given session.
+
+#. Create a ``SessionOptions`` instance by calling the
+   ``SetDefaultTransactionOptions()`` method, passing the
+   ``TransactionOptions`` instance as the parameter.
+
+   You can also specify other session options such as enabling causal
```

Comment:
S: [nit] remove "enabling," since the "such as" should introduce the "other session options"
```suggestion
   You can also specify other session options such as causal
```